_includes/breadcrumbs.html

```diff
@@ -0,0 +1,23 @@
+<section class="solid-breadcrumbs col-xs-12">
```

Comment:
Is the use of `<section>` semantically correct here? Could be perhaps better to just use nav, or simply a div if nav is used in the next part that I commented.

Reply:
I also see that we get the following in the resulting HTML:

```
<section>
  <section>...</section>
  <article>...</article>
  <aside>...</aside>
</section> 
```

I'm thinking perhaps the following is better semantically:


```
<section>
  <nav>...</nav>
  <article>...</article>
  <aside>...</aside>
</section> 
```